sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/implementation/authentication/TokenServiceImpl.java

```diff
@@ -34,11 +34,11 @@ public TokenServiceImpl(String url, ContainerRegistryServiceVersion apiVersion,
             serializerAdapter = JacksonAdapter.createDefaultSerializerAdapter();
         }
 
-        if (apiVersion == null) {
-            apiVersion = ContainerRegistryServiceVersion.getLatest();
-        }
+//        if (apiVersion == null) {
+//            apiVersion = ContainerRegistryServiceVersion.getLatest();
+//        }
```

Comment:
Did the service remove API version?

Reply:
In this swagger version yes. They are working on another version where they will be adding this back.